src/Sylius/Component/Order/Model/OrderItemInterface.php

```diff
@@ -24,6 +24,10 @@ public function getUnitPrice(): int;
 
     public function setUnitPrice(int $unitPrice): void;
 
+    public function getOriginalUnitPrice(): int;
```

Comment:
Isn't that a BC break?

Reply:
Luckily, it's not: https://docs.sylius.com/en/latest/book/organization/backward-compatibility-promise.html#models-model-interfaces